index.html

```diff
@@ -0,0 +1,236 @@
+      </div>
+    
+      <div class="work1" id="w4">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w5">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w6">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+    <section class="AboutMe">
+      <div class="about">
+        <h1>About me</h1>
+        <p>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
+      </div>
+    </section>
+    <section class="myskills">
+      <div class="skills">
+        <div class="languageHeading">
+          <img src="/images/kite.png" alt="" class="kite" />
+          <h1>Languages</h1>
+        </div>
+        <ul class="LanguageSkill">
+          <li>javascript</li>
+          <li>ruby</li>
+          <li>html</li>
+          <li>css</li>
+        </ul>
+        <div class="frameworkheading">
+          <img src="/images/Rectangle 56.png" alt="" />
+          <h1>frameworks</h1>
+        </div>
+        <ul class="frameworkSkill">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+          <li>RSPec</li>
+        </ul>
+        <div class="skillotherheading">
+          <img src="/images/circle.png" alt="" />
+          <h1>Skills</h1>
+        </div>
+        <ul class="skilOther">
+          <li>Codekit</li>
+          <li>github</li>
+          <li>Codepen</li>
+          <li>gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] [OPTIONAL] Instead of having two separate sections, these could just be put into one section as they are all for the same section and then partition the other "skills" section with an article. This would be semantically correct.